.github/workflows/_mac-build.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           auto-update-conda: true
           python-version: 3.8
           activate-environment: build
+          miniconda-version: latest
```

Comment:
Hmm, can you please specify the exact version here? So that we would not accidentally regress when miniconda updates

Reply:
Updated